QUALITY_DECLARATION.md

```diff
@@ -0,0 +1,195 @@
+If the user wants a more detailed report (e.g. the coverage in the different packages), he can follow [this](https://github.com/eclipse-iceoryx/iceoryx/blob/master/CONTRIBUTING.md#coverage-scan) guide to obtain them locally.
+
+### Performance [4.iv]
+
+The most important measurement units for performance testing on iceoryx are the data transfer latency and the CPU load.
+They can be obtained in two ways:
+
+1. [iceperf](https://github.com/eclipse-iceoryx/iceoryx/tree/master/iceoryx_examples/iceperf#iceperf)
+This performance test uses the iceoryx API directly without any indirections to compare different IPC mechanisms on increasing payload sizes. The output is a table linking the payload sizes with the latency in microseconds.
+
+2. [performance_test](https://gitlab.com/ApexAI/performance_test/-/tree/master/)
+The `performance_test` package tests various middleware implementations in ROS using inter-process communication.
+It offers a communication binding for iceoryx and gives information about latency, CPU usage etc.
+Additionally, it is possible to test here the integration of iceoryx + CycloneDDS in ROS using the rmw communication abstraction.
+
+Currently, there are no automatic regression tests in the CI running. The work to integrate this is addressed in [https://github.com/eclipse-iceoryx/iceoryx/issues/137](https://github.com/eclipse-iceoryx/iceoryx/issues/137). The performance tests are currently executed manually before every release that will be integrated into ROS by the maintainers to avoid performance regression.
+
+### Linters and Static Analysis [4.v]
+
+The code formatting is enforced through the CI using `clang-format` and can be used by following this [guide](https://github.com/eclipse-iceoryx/iceoryx/blob/master/CONTRIBUTING.md#coding-style)
+
+The codebase is linted with `clang-tidy` to enforce naming conventions and basic coding rules and checked in the Pull-Requests by review.
+A [static code analysis](https://github.com/eclipse-iceoryx/iceoryx/blob/master/CONTRIBUTING.md#static-code-analysis) is done regularly with the Axivion Suite to check against a defined [ruleset](https://github.com/eclipse-iceoryx/iceoryx/blob/master/tools/axivion/axivion_config.json), but the results are not publicly accessible due to licensing.
+
+## Dependencies [5]
+
+### Direct Runtime ROS Dependencies [5.i]
+
+As an external dependency, there are no ROS dependencies in `iceoryx_hoofs`, `iceoryx_posh`, and `iceoryx_binding_c`.
+
+### Optional Direct Runtime ROS Dependencies [5.ii]
+
+As an external dependency, there are no ROS dependencies in `iceoryx_hoofs`, `iceoryx_posh`, and `iceoryx_binding_c`.
+
+### Direct Runtime non-ROS Dependency [5.iii]
+
+The only runtime dependency is the [Access Control List](https://en.wikipedia.org/wiki/Access-control_list) support that can be installed within the system packages. The underlying filesystem needs to support ACL.
```

Comment:
I'm not sure if we need to state that a POSIX system is required, e.g. `Beside a POSIX compliant OS, the only runtime ...` since we also need `pthread`, etc.